monitoring/ceph-mixin/prometheus_alerts.yml

```diff
@@ -704,15 +704,26 @@ groups:
             rate of the past 48 hours.
 
       - alert: CephNodeInconsistentMTU
-        expr: node_network_mtu_bytes{device!="lo"} * (node_network_up{device!="lo"} > 0) != on() group_left() (quantile(0.5, node_network_mtu_bytes{device!="lo"}))
+        expr: |
+          node_network_mtu_bytes * (node_network_up{device!="lo"} > 0) ==
+            scalar(
+              max by (device) (node_network_mtu_bytes) !=
+                quantile by (device) (.5, node_network_mtu_bytes)
+            )
+          or
+          node_network_mtu_bytes * (node_network_up{device!="lo"} > 0) ==
+            scalar(
+              min by (device) (node_network_mtu_bytes) !=
+                quantile by (device) (.5, node_network_mtu_bytes)
+            )
```

Comment:
```suggestion
        expr: |
          node_network_mtu_bytes * (node_network_up{device!="lo"} > 0) ==
            scalar(
              max by (device) (node_network_mtu_bytes * (node_network_up{device!="lo"} > 0)) !=
                quantile by (device) (.5, node_network_mtu_bytes * (node_network_up{device!="lo"} > 0))
            )
          or
          node_network_mtu_bytes * (node_network_up{device!="lo"} > 0) ==
            scalar(
              min by (device) (node_network_mtu_bytes * (node_network_up{device!="lo"} > 0)) !=
                quantile by (device) (.5, node_network_mtu_bytes * (node_network_up{device!="lo"} > 0))
            )
```

Reply:
Thanks @p-se for this PR. Am in process of writing the promtool test for this, I think we need to consider only the devices that are in up state in calculations as well as I found some use cases failing becaiuse of this.